Sources/SwiftInspectorVisitors/PropertyVisitor.swift

```diff
@@ -74,23 +76,24 @@ public final class PropertyVisitor: SyntaxVisitor {
   // MARK: Private
 
   private func findModifiers(from node: VariableDeclSyntax) -> PropertyInfo.Modifier {
```

Comment:
A couple of interesting findings that I had.
<img width="1193" alt="Screen Shot 2021-09-15 at 1 11 25 PM" src="https://user-images.githubusercontent.com/1791049/133502723-b2982f01-658c-4d55-b93d-4a89eaf541c7.png">
<img width="1199" alt="Screen Shot 2021-09-15 at 1 11 42 PM" src="https://user-images.githubusercontent.com/1791049/133502739-62c250dd-2a1a-4980-a4ec-b8b652537c80.png">